src/components/DocsNavs.re

```diff
@@ -472,18 +472,18 @@ module SideNav = {
           title={intl->Intl.formatMessage(howZkAppUI)}
           slug="how-to-write-a-zkapp-ui"
           />
-          <Item
-            title={intl->Intl.formatMessage(zkAppsForEthereumDevelopers)}
-            slug="zkapps-for-ethereum-developers"
-          />
-          <Item
-            title={intl->Intl.formatMessage(simpleAnonymousMessageBoardTutorial)}
-            slug="simple-anonymous-message-board-tutorial"
-          />
           <Item
           title={intl->Intl.formatMessage(advancedConcepts)}
           slug="advanced-concepts"
```

Comment:
`advanced-snarkyjs`. And any links to it too

Reply:
Done